src/plugins/data_source/config.ts

```diff
@@ -13,7 +13,7 @@ const WRAPPING_KEY_SIZE: number = 32;
 
 export const configSchema = schema.object({
   enabled: schema.boolean({ defaultValue: false }),
-  defaultCluster: schema.boolean({ defaultValue: true }),
+  hideLocalCluster: schema.boolean({ defaultValue: false }),
```

Comment:
the default value was changed, is there testing cover this default behavior changes? thanks

Reply:
Thanks for the comment.

Yes, this testing already been covered under `"src/plugins/data/server/search/opensearch_search/opensearch_search_strategy.test.ts"`